### PR TITLE
Use Debian12 base containers in export flows

### PR DESCRIPTION
**Pull request by barharel-google**

### CODE REVIEW

File: gce_ovf_export.Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM launcher.gcr.io/google/debian11
+FROM launcher.gcr.io/google/debian12
```
Comment (aryeyur) [inline gce_ovf_export.Dockerfile:15]: Debian 12 is bookworm, I assume the GCSFUSE_REPO needs to be adjusted accordingly (11 is bullseye)

Comment (barharel-google) [reply gce_ovf_export.Dockerfile:15]: Done, thanks

**Pull request merged**